File: jogos_educativos.java
import java.util.Scanner;

public class jogos_educativos {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Total de jogos vendidos no mês:");
        int jogosVendidos = scanner.nextInt();
        
        double precoJogo = 19.90;
        double totalVendas = jogosVendidos * precoJogo;
        int bonusVendas = jogosVendidos / 15;
        double valorBonus = totalVendas * 0.08 * bonusVendas;
        double salario = totalVendas * 0.5 + valorBonus;
        
        System.out.printf("Valor total das vendas: %.2f\nBônus: %.2f\nSalário: %.2f\n", totalVendas, valorBonus, salario);
    }
}

File: jogos_educativos.c
#include <stdio.h>

int main() {
    int jogosVendidos;
    printf("Total de jogos vendidos no mes:\n");
    scanf("%d", &jogosVendidos);
    
    double precoJogo = 19.90;
    double totalVendas = jogosVendidos * precoJogo;
    int bonusVendas = jogosVendidos / 15;
    double valorBonus = totalVendas * 0.08 * bonusVendas;
    double salario = totalVendas * 0.5 + valorBonus;
    
    printf("Valor total das vendas: %.2f\nBonus: %.2f\nSalario: %.2f\n", totalVendas, valorBonus, salario);
    
    return 0;
}
